### DISCUSSION TITLE
The Metro Project

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1133] [Maps and Content Showcase]
This is a idea i've been working on for the last few days, of making a map for the tess engine, with a a lot of models replacing geometry when it's necessary.
What do i plan to accomplish with this?
1: It's mostly a learning experience for myself.
2: try to get to make a very beautiful map, lighting/modeling wise/push tess further than im used to.

So far, i made these models for this map:
Train track
metro pillar 1/large
metro pillar 1-2/large
metro roof 1 (untextured)
waiting chair
underground power cable.

I don't know when i will finish it, so we will see how this turns out as time goes on.

~~ timeline ~ 
Rough first version of the map, this only had the tracks and some basic lighting, made the track and cable models.
![20171127165808](https://user-images.githubusercontent.com/24359960/36062155-8d582cc6-0e4d-11e8-8618-d730e40cf5d1.png)

Came back to it after a month or so, started to do a rough shape of how it will be.
![20180207235807](https://user-images.githubusercontent.com/24359960/36062125-0d612df6-0e4d-11e8-8ed6-046e6138d5e6.png)

The waiting place went said goodbye, and expanded the map a litttle.
![20180208225542](https://user-images.githubusercontent.com/24359960/36062130-33782238-0e4d-11e8-8e37-b5ebd6ba93df.png)
![20180208225655](https://user-images.githubusercontent.com/24359960/36062141-5654743c-0e4d-11e8-968a-5aa51d7474f7.png)

Added some sunlight, which got me surprised, as it made the environment way more alive.
![20180208232326](https://user-images.githubusercontent.com/24359960/36062174-ddf17fb6-0e4d-11e8-8bad-fe8e85f4a5bb.png)
![20180208232337](https://user-images.githubusercontent.com/24359960/36062175-de10b624-0e4d-11e8-8970-2e4c3a53dbd8.png)
![20180208232055](https://user-images.githubusercontent.com/24359960/36062176-de2f4936-0e4d-11e8-8335-b934ffc0ae83.png)

Some pillar models.
![20180209185811](https://user-images.githubusercontent.com/24359960/36062192-16efac2a-0e4e-11e8-882f-137ac8dad85d.png)

Tried to make a roof model (untextured) and placed it above the train lines.
![20180209213527](https://user-images.githubusercontent.com/24359960/36062199-3616dbfa-0e4e-11e8-9082-3d4c5ff76845.png)

And that's it for now.
Im thinking of making an actual map with these assets later, so who knows, maybe this map itself will transform in a playable layout?
Will update when new things get added/modeled ,

**Comment by qreeves** [comment]
Yeah I've seen this setting tried quite a few times, but the assets do really help sell the aesthetic. As you noted, the trick is trying to find a playable layout. Looks good though 😛

**Comment by SniperGoth** [comment]
Thanks, i never went to a metro in person, so getting the look right has been quite a challenge, hah.
Anyway, a new asset was made, m-light1.

It requires envmaps to work as intended, giving it a metallic-look. 
![20180211134144](https://user-images.githubusercontent.com/24359960/36077497-c915a5f8-0f52-11e8-9518-5359751263e4.png)
![20180211172934](https://user-images.githubusercontent.com/24359960/36077519-087a61f2-0f53-11e8-9b39-46b88841c726.png)

There's also a non-env version of it : 
![20180211174539](https://user-images.githubusercontent.com/24359960/36077550-6e0c0bce-0f53-11e8-8442-560a4c058a5f.png)
![20180211174533](https://user-images.githubusercontent.com/24359960/36077553-75d34886-0f53-11e8-918a-c17054c185e4.png)
![20180211174245](https://user-images.githubusercontent.com/24359960/36077554-78edd8a6-0f53-11e8-86c4-9a6b1ef6550a.png)

Im gonna release every asset when the map gets finished, so everyone can use them later. ;)

**Comment by SniperGoth** [comment]
Layout is finally starting to take form.
It's going to be very simple, but after all, this is supposed to be more of an tech demo of sorts, so i guess it's passable .
Also, the bars of the roof model make use of envmaps now.
![20180217113421](https://user-images.githubusercontent.com/24359960/36341522-bbb1de8a-13d6-11e8-9e08-0dc4c1ec8d23.png)
![20180217113439](https://user-images.githubusercontent.com/24359960/36341523-bbdcb51a-13d6-11e8-9e0e-e5d19262a3da.png)
![20180217113506](https://user-images.githubusercontent.com/24359960/36341524-bbfb3b98-13d6-11e8-8e62-ef5f8a7100a3.png)
